### DISCUSSION TITLE
lockR.lock

### DISCUSSION THREAD

**Discussion by Vahaggn** [minio/minio#15324] [Q&A] [answered]
Hi Every one i have a problem with my MinIO Cluster
i have 4 Servers of MinIO which each of them have 1 Disk- and i have 14 Buckets
when i enable trace the logs in internal mode i found some errors like this
lockR.lock 403 <MINIO SERVER ADDRESS>
any idea?

**Comment by harshavardhana** [comment, marked as answer]
They are not errors those are internal calls.
